js/copy-service/Parser/Parser.test.js

```diff
@@ -283,7 +299,7 @@ describe('Parser', () => {
           describe('when a tree contains copy nested inside HTML tags', () => {
             test('parses the nested copy successfully', () => {
               const initialTree = {
-                tags: '<strong><em>${hello}\n#{sub}</em></strong>'
+                tags: '<strong><em>${hello}\n#{sub}\b</em></strong>'
```

Comment:
Let's add text after the `\b` to validate sibling behavior and so this example is not completely syntax.